docs/src/3_Automation/04_Powers.yml

```diff
@@ -1,641 +1,740 @@
+        - "Ace of Spades: As a Minor Action, you can strike an enemy with an automatic hit with a Power of your choosing"
+    Type: Passive
+    XP: 2
+    Prereq:
+        Level: 3
+    Tags: Fate Cards
 
 Barrier:
-  Category: Magic
-  Subcategory: None
-  Description: Your skills allow you to deploy temporary impromptu walls
-  Mechanic: You create a wall 3 or 5 spaces wide. You may dismiss this wall at any time as a Free Action. This wall is impervious to standard attacks, but may be worn down over time at the GM's discretion.
-  Type: Major
-  XP Cost: 2
-  PP Cost: [1,2]
-  Prereq Role: [Support, Caster]
-  Tags:
+    Category: 
+        - Magic
+    Description: Your skills allow you to deploy temporary impromptu walls
+    Mechanic: You create a wall 3 or 5 spaces wide. You may dismiss this wall at any time as a Free Action. This wall is impervious to standard attacks, but may be worn down over time at the GM's discretion.
+    Type: Major
+    XP: 2
+    PP: [1,2]
+    Prereq:
+        Role: [Support, Caster]
 
 Illusion:
-  Category: Magic
-  Subcategory: None
-  Description: Your skills let you conjure illusory effects. 
-  Mechanic: You make an illusory intangible visual (no larger than 1 space) or auditory effect that you've heard before. To determine if it's real, a character makes a Contested Conviction Check. For 2 PP, you can generate both visual and auditory effects, and the check to verify draws with the Lower Hand.
-  Type: Major
-  XP Cost: 2
-  PP Cost: [1,2]
-  Prereq Role: Caster
-  Tags:
+    Category: 
+        - Magic
+    Description: Your skills let you conjure illusory effects. 
+    Mechanic: You make an illusory intangible visual (no larger than 1 space) or auditory effect that you've heard before. To determine if it's real, a character makes a Contested Conviction Check. For 2 PP, you can generate both visual and auditory effects, and the check to verify draws with the Lower Hand.
+    Type: Major
+    XP: 2
+    PP: [1,2]
+    Prereq:
+        Role: Caster
 
 Speedy:
-  Category: Movement
-  Subcategory: Speed
-  Description: You're quicker than the rest.
-  Mechanic: When you take this Power, your combat speed increases by 2 squares.
-  Type: Passive
-  XP Cost: 1
-  Tags: Combat
+    Category: 
+        - Movement
+        - Speed
+    Description: You're quicker than the rest.
+    Mechanic: When you take this Power, your combat speed increases by 2 squares.
+    Type: Passive
+    XP: 1
+    Tags: Combat
 
 Stealthy Surprise:
-  Category: Movement
-  Subcategory: Stealth
-  Description: An undetected assailant is particularly deadly.
-  Mechanic: You may Hide as a Minor Action.
-  Type: Minor
-  XP Cost: 1
-  Prereq Skill: Stealth > 1
-  Tags: Stealth
+    Category: 
+        - Movement
+        - Stealth
+    Type: Minor
+    Description: An undetected assailant is particularly deadly.
+    Mechanic: Hide.
```

Comment:
I think my intent here was 'you may hide as a minor action' rather than 'deal additional damage when hidden'

Reply:
Oh ok, could we revise to the following? 

Since it's already a Minor I think that works to convey it requires a Minor Action to attempt to Hide (still make a Stealth Check).
```suggestion
    Mechanic: You attempt to hide from nearby creatures.
```